src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Model/EnterpriseUserCredentials.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Altinn.Platform.Authentication.Model
+{
+    /// <summary>
+    /// Handles credentials for EnterpriseUsers
+    /// </summary>
+    public class EnterpriseUserCredentials
+    {
+        /// <summary>
+        /// Gets or sets the user name
+        /// </summary>
+        public string UserName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the password
+        /// </summary>
+        public string Password { get; set; }
+
+        /// <summary>
+        /// Gets or sets the organization number
+        /// </summary>
+        public string OrganizationNumber { get; set; }
```

Comment:
We have the joy of working with british spelling in Altinn 3, so it should be OrganisationNumber to match the other org properties.

Reply:
Or even better, use `OrgNumber`. Then the entire spelling-problem goes away, and code becomes both shorter and more readable. It's even in the scrabble dictionary now :) https://www.wordgamedictionary.com/dictionary/word/org/

Reply:
The EnterpriseUserCredentials-class maps directly to equivalent class on sblbridge so it seems it cannot be renamed. It results in "Bad request" from sblbridge.